### ISSUE TITLE
proxy list is resorted on activation

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-ios#2414]
when tapping a proxy in a list of 2+ proxies, the list gets resorted, which makes it hard to follow which proxies were tested and which not. 

ftr, some days ago, the list was stable while the ViewController was open and got only resorted when the ViewController was closed and re-opened

**Comment by zeitschlag** [comment]
Notes to self: Seems to be caused by the connectivity-change-notification-handling.
